chain/ethereum/src/adapter.rs

```diff
@@ -220,6 +306,8 @@ pub struct EthereumLogFilter {
     /// Event sigs with no associated address, matching on all addresses.
     /// Maps to a boolean representing if a trigger requires a transaction receipt.
     wildcard_events: HashMap<EventSignature, bool>,
+    /// Events with any of the topic filters set
+    events_with_topic_filters: HashMap<EventSignatureWithTopics, bool>,
```

Comment:
do you use the bool here for anything? Should this be a hashset?

Reply:
seems to be used to mean "requires_receipt", can you add some clarification around it with a comment?